README.md

```diff
@@ -7,9 +7,17 @@ objectives, and comply with regulations.
 
 <!-- toc -->
 * [Install](#install)
+  * [Requirements](#requirements)
+    * [Networking](#networking)
+    * [Supported OS & Architectures](#supported-os-and-architectures)
+    * [Supported Node.js Versions](#supported-nodejs-versions)
+  * [Methods](#methods)
+    * [Using npm](#using-npm)
+    * [Using Tarballs](#using-tarballs)
 * [Usage](#usage)
-* [Environment Variables](#environment-variables)
+* [Login](#login)
 * [Command Topics](#command-topics)
+* [Environment Variables](#environment-variables)
```

Comment:
This should be fine because I am pretty sure toc doesn't get changed now by the command now but I will double check